astropy/visualization/mpl_style.py

```diff
@@ -61,13 +61,15 @@
     'axes.labelsize': 'large',
     'axes.labelcolor': '#FFFFFF',
     'axes.axisbelow': True,
-    'axes.color_cycle': ['#348ABD',   # blue
-                         '#7A68A6',   # purple
-                         '#A60628',   # red
-                         '#467821',   # green
-                         '#CF4457',   # pink
-                         '#188487',   # turquoise
-                         '#E24A33'],  # orange
+    'axes.color_cycle': [
+        '#E24A33',   # orange
+        '#348ABD',   # blue
+        '#467821',   # green
+        '#A60628',   # red
+        '#7A68A6',   # purple
+        '#CF4457',   # pink
+        '#188487'    # turquoise
+    ],
 
```

Comment:
Since the docs have their own style set below, I think this change should be reverted.